init/Makefile

```diff
@@ -1,11 +1,20 @@
-CC       = gcc
-C_FLAGS := -Wall -Wextra -Werror -O2
+
+ARCH_aarch64	       = arm64
+C_FLAGS 	      := -Wall -Wextra -Werror -O2
+CC 		      := gcc
+DEFAULT_CROSS_COMPILER = /usr/bin/aarch64-linux-gnu-
+
+ifeq ($(ARCH),$(ARCH_aarch64))
```

Comment:
In the main Makefile, the one from above, ARCH should be passed in "make -C $(PWD)/init" via ARCH=$(ARCH).

And should update here as well, to check for the HOST_MACHINE arch and set cross compile if it is not Arm64.

Reply:
yes, I am working on a fixup there too :D